app/code/community/Razorpay/Payments/Model/Paymentmethod.php

```diff
@@ -174,6 +174,26 @@ public function _getChannel()
         return sprintf(self::CHANNEL_NAME, $edition, Mage::getVersion(), self::VERSION);
     }
 
+    protected function hash_equals($str1, $str2)
```

Comment:
Add a condition `if (function_exists('hash_equals'))` then use the native function else use your definition